### ISSUE TITLE
Image Block: Inconsistent image resizing

### ISSUE THREAD

**Issue by CreativeDive** [WordPress/gutenberg#17685]
Hey, I can scale the image block wider than the content width. This is inconsistent. The image block should never be wider than the content width.

![image-block-resize](https://user-images.githubusercontent.com/31563167/65985937-2985e400-e483-11e9-89d3-2f71daf4fc68.gif)

**Comment by CreativeDive** [comment]
The same point to manually set the image dimensions (editor sidebar) wider than the content width. That should not be possible.

**Comment by CreativeDive** [comment]
And an other design bug by set the image height manually to a large value like 2000. This displays the image block incorrectly.

![image-height](https://user-images.githubusercontent.com/31563167/65989517-293d1700-e48a-11e9-9720-3ab34f1bc2f8.gif)

**Comment by skorasaurus** [comment]
very similar to #12168

**Comment by CreativeDive** [comment]
@skorasaurus Yes I confirm, one of this issues is similar. But there is also an **other issue by set the image height**. Therefore do not close this issue as a duplicate please.

**Comment by jordesign** [comment]
With much more discussion (and more recently) I'm going to close this issue as it will be resolved by any fix for https://github.com/WordPress/gutenberg/issues/12168
